plugins/ExhibitBuilder/helpers/ExhibitPageFunctions.php

```diff
@@ -312,13 +312,13 @@ function exhibit_builder_page_summary($exhibitPage = null)
         $exhibitPage = get_current_record('exhibit_page');
     }
 
-    $html = '<li>'
-          . '<a href="' . exhibit_builder_exhibit_uri(get_current_record('exhibit'), $exhibitPage) . '">'
+    $html = '<li class="exhibit-nav-list-items">'
```

Comment:
Is there a way we can change the html (line 315-316 and 321) from the theme?

Reply:
I believe it is good practice to decouple the HTML from the CSS by using these added classes. The HTML exists in this PHP file so I added the class that the CSS addresses in the theme here.